src/sentry/testutils/pytest/peanutbutter.py

```diff
@@ -0,0 +1,82 @@
+# Fixtures used to interact with a test peanutbutter server
+
+from os import environ
+
+# import grpc
+import pytest
+
+from sentry.runner.commands.devservices import get_docker_client
+
+PEANUTBUTTER_TEST_IMAGE = environ.get(
+    "PEANUTBUTTER_TEST_IMAGE",
+    "us.gcr.io/sentryio/peanutbutter:latest",
+)
+
+
+def _peanutbutter_server_container_name():
+    return "sentry_test_peanutbutter_server"
+
+
+# FIXME: copied from `relay.py`, but maybe this should be a generic util?
```

Comment:
relay bypasses the normal way that devservices normally works -- I'd rather this stick to the normal way and just delete this file (since the container would already be available if it were started and not need to be started on-demand which adds a bunch of complexity for not much (zero?) benefit)